examples/mobile/UIComponents/components/list/multiple-selection-list.html

```diff
@@ -13,8 +13,9 @@
 	<div class="ui-page" id="checkbox-list-selected-demo">
 		<header>
 			<div class="ui-appbar-left-icons-container">
-				<label>All
-					<input class="ui-checkbox-all" type="checkbox"/>
+				<label class="ui-checkbox-all">
```

Comment:
How about changing the name of the 'ui-checkbox-all' to the 'ui-checkbox-label'?
Wouldn't it be good to consider other names as well?

Reply:
I didn't think too much about it. This can be named also "ui-checkbox-label-all". This is rather specific component for only one purpose.

Reply:
In my opinion, it is better to use it in a broader name than a specific purpose in terms.
How about your opinions @lslachciak, @singa2000 ?

Reply:
for <li> we have classes like ui-li-has-checkbox ui-li-divider
so to be common in style we should have "label" in name
so I think this should be called according to the ui-label-* schema
e.g.
ui-label-select-all

Reply:
Okay,, `ui-label-select-all` seems to be more reasonable.